src/components/auth/AuthButton.js

```diff
@@ -4,22 +4,26 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { withRouter } from 'react-router-dom';
 
-import { logUserOut } from 'console/state/auth/actions';
-import { getUserProfile } from 'console/state/auth/selectors';
+import { logUserOut, setAuthStarted } from 'console/state/auth/actions';
+import { getAuthStarted, getUserProfile } from 'console/state/auth/selectors';
 import { startAuthenticationFlow } from 'console/utils/auth0';
 
 @connect(
   (state, props) => ({
     userProfile: getUserProfile(state),
+    authStarted: getAuthStarted(state),
   }),
   {
     logUserOut,
+    setAuthStarted,
   },
 )
 @withRouter
```

Comment:
dummy question; this passing in a bunch of extra props such as `location`, why don't we list all of those in the static `propTypes` below?

Reply:
this will be a non-issue once my router changes land. we will no longer be doing this.